runtime/mlBytes.js

```diff
@@ -425,9 +425,14 @@ MlBytes.prototype.toString = function(){
     }
     this.t = 8; /*BYTES | NOT_ASCII*/
   case 8: /*BYTES | NOT_ASCII*/
-    return caml_utf16_of_utf8(this.c);
+    return this.c;
```

Comment:
What is the type of this.c?

Reply:
See the top of that file. It describes the memory representation of mlBytes